tests/conftest.py

```diff
@@ -57,16 +57,32 @@ def get_user(email):
         return openreview.Client(baseurl = 'http://localhost:3000', username = email, password = Helpers.strong_password)
 
     @staticmethod
-    def await_queue(super_client=None):
+    def await_queue(super_client=None, jobNames=None, api_version=1):
+        if jobNames is None:
+            jobNames = set([
+                'emailQueueStatus',
+                'jsEntityQueueStatus',
+                'pyEntityQueueStatus',
+                'gatherEmailsQueueStatus',
+                'jsDateProcessQueueStatus',
+                'pyDateProcessQueueStatus',
+                'fileUploaderQueueStatus',
+                'fileDeletionQueueStatus'
+            ])
         if super_client is None:
-            super_client = openreview.Client(baseurl='http://localhost:3000', username='openreview.net', password=Helpers.strong_password)
-            assert super_client is not None, 'Super Client is none'
+            if api_version == 1:
```

Comment:
I don't think this is needed, you can pass the api 2 client to this function, that is what I usually do.

Reply:
Anyway I think I fixed the tests without using these changes, can I revert them?

Reply:
Sure, if they are not needed we can revert them.